packages/aws-cdk-lib/aws-cognito/test/user-pool-idps/apple.test.ts

```diff
@@ -90,8 +90,8 @@ describe('UserPoolIdentityProvider', () => {
         keyId: 'CDKKEYCDK1',
         privateKey: 'PRIV_KEY_CDK',
         attributeMapping: {
-          familyName: ProviderAttribute.APPLE_LAST_NAME,
-          givenName: ProviderAttribute.APPLE_FIRST_NAME,
+          familyName: ProviderAttribute.APPLE_FIRST_NAME,
```

Comment:
This is not part of this PR but it won't pass if unchanged so we have to update it. I am not sure if relevant code has to be fixed but we don't fix that in this PR.

Reply:
Instead of changing this, I think you should change the verification step. Family name is last name.

Reply:
yes you are right!!